packages/feedback-spa/src/app/modules/feedback/home/feedback-home.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
 import { FeedbackService } from '../feedback.service';
```

Comment:
Wait a minute. Where is this file coming from?
I don't see that file anywhere in the directory, or the entire package.

Reply:
That file is there in my directory structure see bellow screenshot.
![image](https://user-images.githubusercontent.com/12980398/79428716-31067800-7fe4-11ea-9326-b511a25504ad.png)

Reply:
Bellow is the working screenshot.

![image](https://user-images.githubusercontent.com/12980398/79430880-387b5080-7fe7-11ea-873c-a8668fc5ce18.png)

Reply:
Where is that data coming from?

Reply:
@deshmukhmayur  i updated directory structure and and also move mock file.
Please have a another look.

Reply:
For now testing purpose i am using one-portal backend coz for one platform backend is not ready yet.